templates/build_passed.md.erb

```diff
@@ -1 +1 @@
-Good news! Your pull request passed all our automated tests. Our volunteers will try to take a look soon.
+Yay! Your pull request passed all our automated tests. We'll try to take a look soon.
```

Comment:
It'd take out "try" here and below. "We'll take a look soon." "We'll get in touch soon"